gtfs-realtime/CHANGES.md

```diff
@@ -68,6 +68,8 @@ To allow producers to add custom information to a GTFS Realtime feed, we will ta
 
 When possible we should avoid extensions and add fields useful to many agencies to the main specification to avoid fragmentation and extra work for consumers to support various extensions to the spec.  Before requesting an extension id, producers should propose adding the field to the specification (see [Adding new fields to GTFS Realtime](#adding-new-fields-to-gtfs-realtime))
 
+The extension IDs >= 9000 are reserved for private use by GTFS-rt producers. These IDs should only be used to convey information internally to your organization. If you need to publish information in the GTFS-rt you produce, it is encouraged to ask for a new extension ID (see below). 
+
```

Comment:
Could you please add an entry to the extension registry table at the bottom of the page? 

>|>= 9000|RESERVED - INTERNAL USE ONLY|[GTFS community](https://groups.google.com/forum/#!forum/gtfs-realtime)|[Group discussion](https://github.com/google/transit/pull/178/)|

Reply:
Sure, it is done 👍